### PR TITLE
Work Complete

### PR DESCRIPTION
**Pull request by AvigailGellis**
Hi! Great biz scenario! Loved the clients:) Sorry only got it done now, I started bofore I was leaving for the week. 
Here is all my work, comment in the pull request if something needs to be changed.

### CODE REVIEW

File: PersonalTrainingDB/3 reports.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+use PersonalTrainingDB
+go
+
+--I want to see a list of all my current clients presented as follows:
+    --last name, first name, starting weight, current weight, weight lost so far, goal weight if applies,
+    --trainer, how long they have been training with us, ordered by newest clients to oldest clients
+select c.LastName, c.FirstName, c.StartWeight, c.CurrentWeight, WeightLost = (c.StartWeight - c.CurrentWeight), 
+    GoalWeight = case when ClientsGoal like '%Goal Weight%' then ClientsGoal else '' end, c.PrimaryTrainer, YearsSinceBeganTraining = concat(datediff(year, c.SignupDate, getdate()), ' Years')
+from Client c
+order by datediff(month, c.SignupDate, getdate()) desc
```
Comment (brachagreenberg) [inline PersonalTrainingDB/3 reports.sql]: It seems like the order by is incorrect, spec says to show the clients newest to oldest

File: PersonalTrainingDB/3 reports.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+--Show me the min max and average weight loss per trainer
+select c.PrimaryTrainer, MinWeightLoss = min(c.StartWeight - c.CurrentWeight), MaxWeightLoss = max(c.StartWeight - c.CurrentWeight), AvgWeightLoss = avg(c.StartWeight - c.CurrentWeight)
+from Client c 
+group by c.PrimaryTrainer
+--Show me a list how $ much each client owes greatest to least. include phone numbers in this list.
+select c.FirstName, c.LastName, c.BalanceDue, c.PhoneNumber
+from Client c
```
Comment (brachagreenberg) [inline PersonalTrainingDB/3 reports.sql]: it seems like you are missing the order by, spec says to show how much $ each client owes greatest to least

**Review by brachagreenberg — CHANGES_REQUESTED** [review]
Thank you so much! Its nearly perfect, see comments

File: PersonalTrainingDB/3 reports.sql (hunk 1 above)
Comment (AvigailGellis) [reply PersonalTrainingDB/3 reports.sql]: fixed

File: PersonalTrainingDB/3 reports.sql (hunk 2 above)
Comment (AvigailGellis) [reply PersonalTrainingDB/3 reports.sql]: fixed

File: PersonalTrainingDB/3 reports.sql (hunk 1 above)
Comment (brachagreenberg) [reply PersonalTrainingDB/3 reports.sql]: perfect

File: PersonalTrainingDB/3 reports.sql (hunk 2 above)
Comment (brachagreenberg) [reply PersonalTrainingDB/3 reports.sql]: perfect

**Pull request closed without merging**